src/Pages/LoginPage/Components/LoginForm.jsx

```diff
@@ -0,0 +1,140 @@
+import { useState } from "react";
+import { Link, useNavigate, useLocation } from "react-router-dom";
+import { useModal, useAuth, useAlert } from "../../../CustomHooks/";
+import axios from "axios";
+
+export const LoginForm = (props) => {
+  let navigate = useNavigate();
+  const { authDispatch } = useAuth();
+  const { setShowModal } = useModal();
+  const { pathname } = useLocation();
+  const { setShowAlert } = useAlert();
+  const [error, setError] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [loginDetails, setLoginDetails] = useState({
+    email: "",
+    password: "",
+  });
+
+  const loginInputHandler = (e) => {
+    setLoginDetails({
+      ...loginDetails,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const formSubmitHandler = async (e) => {
+    e.preventDefault();
+    const { email, password } = loginDetails;
+    setError("");
+    try {
+      const response = await axios.post("/api/auth/login", {
```

Comment:
This could have been a different function like login services and you can import that here to make it more modular, so make a folder called services and inside that add that. The code is perfectly fine and working but this was the common folder structure trend I saw.

Reply:
Makes sense. I thought about this too while creating this function but ended up not moving it. Will do it, thanks.